### ISSUE TITLE
Webserver not opening

### ISSUE THREAD

**Issue by pechhenka** [plan-player-analytics/Plan#1392]
### Describe the issue
Hello!
I installed this plugin and it even works correctly, but I just can not open link Webserver. The plugin generates a link, but when I click on it the browser says that I can’t access the site

### Server information
Server:
- Bukkit&Spigot
- Minecraft version 1.15.2

Player Analytics
- Version: 5.1 build 497
- Update Available: No
- Active Database: SQLite (OPEN)
- Connected to Proxy: No

### Exceptions & Other Logs
Log when Plan start:
```
[Plan] Processing critical unprocessed tasks. (0)
[Plan] Processing complete.
[Plan] Webserver has been disabled.
[Plan] Player Analytics Disabled.

[Plan] Locale: 'English' by Rsl1122
[Plan] You're using the latest version.
[Plan] SQLite-database connection established.
[Plan] WebServer: Certificate KeyStore File not Found: plugins/Plan/Cert.jks
[Plan] WebServer: No Certificate -> Using HTTP-server for Visualization.
WebServer: User Authorization Disabled! (Not secure over HTTP)
[Plan] Webserver running on PORT 8804 (http://148.251.187.71:8804)
[Plan] Registered extension: Essentials
[Plan] Downloading GeoLite2 requires accepting GeoLite2 EULA - see 'Data_gathering.Accept_GeoLite2_EULA' in the config.
[Plan] Fallback: using IP2C for Geolocation (doesn't support IPv6).
[Plan] Registered extension: LuckPerms
[Plan] Registered extension: Economy (Vault)
[Plan] Registered extension: Permission Groups (Vault)
[Plan] Player Analytics Enabled. (168 ms)
Reload Complete
```

### Additional information
My Plan config(Wevserver):
```
Webserver:
    Port: 8804
    Alternative_IP: true
        # %port% is replaced automatically with Webserver.Port
        Address: 148.251.187.71:%port%
    # InternalIP usually does not need to be changed, only change it if you know what you're doing!
    # 0.0.0.0 allocates Internal (local) IP automatically for the WebServer.
    Internal_IP: 0.0.0.0
    Security:
        SSL_certificate:
            KeyStore_path: Cert.jks
            Key_pass: default
            Store_pass: default
            Alias: alias
        # HTTPS is required for Login.
        Disable_authentication: false
        # Cross-Origin Resource Sharing (Requests from non-Plan web pages)
        # https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
        CORS:
            Allow_origin: "*"
    Disable_Webserver: false
    External_Webserver_address: http://148.251.187.71
```
Help me please. Maybe I just filled out the config file incorrectly or my host does not support Webserver.
I keep my server on hosting-minecraft.ru

**Comment by Rsl1122** [comment]
The port 8804 is closed
![Port is closed](https://user-images.githubusercontent.com/24460436/78397832-9e84d280-75fa-11ea-8f1f-27f8a26273b6.png)
https://www.yougetsignal.com/tools/open-ports/

**Comment by pechhenka** [comment]
Well, it looks like the problem was indeed in a closed port. Here is my solution to who might run into the same problem:
1. I wrote to tech support asking me to open port 8804. The answer came to me that they “opened” it, although this is not so.
2. Although they did not open it, new ports appeared in my control panel (previously there was only one - the main one)
3. I just checked it for openness (it was open) and replaced him instead of 8804 in the config file.
Done!
